doc/sphinx-guides/source/versions.rst

```diff
@@ -1,10 +1,10 @@
 :orphan:
 .. _guides_versions:
 
-Dataverse Documentation Versions
+Dataverse Software Documentation Versions
 ================================
```

Comment:
Additional header format characters required to match the length of the header text above.

Reply:
fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc